config/features/config.go

```diff
@@ -41,6 +42,7 @@ type Flags struct {
 	WriteSSZStateTransitions            bool // WriteSSZStateTransitions to tmp directory.
 	EnablePeerScorer                    bool // EnablePeerScorer enables experimental peer scoring in p2p.
 	DisableReorgLateBlocks              bool // DisableReorgLateBlocks disables reorgs of late blocks.
+	EnableLightClientEvents             bool // EnableLightClientEvents enables light client events to be emitted by the beacon node.
```

Comment:
Can you rename this to `EnableLightClient` please? We will probably want to use this flag in upcoming PRs too for things other than events e.g. saving to the DB. Therefore a more general name will be better suited.

Reply:
Patched.